datadog_checks_dev/datadog_checks/dev/tooling/commands/meta/snmp/generate_profile.py

```diff
@@ -0,0 +1,624 @@
+    SYNTAX      EntPhysicalContainsEntry
+    MAX-ACCESS  not-accessible
+    STATUS      current
+    DESCRIPTION
+            "A single container/'containee' relationship."
+    INDEX       { entPhysicalIndex, entPhysicalChildIndex }
+    ::= { entPhysicalContainsTable 1 }
+    ```
+    or its json dump, where `INDEX` is replaced by indices
+    ```json
+    "entPhysicalContainsEntry": {
+        "name": "entPhysicalContainsEntry",
+        "oid": "1.3.6.1.2.1.47.1.3.3.1",
+        "nodetype": "row",
+        "class": "objecttype",
+        "maxaccess": "not-accessible",
+        "indices": [
+          {
+            "module": "ENTITY-MIB",
+            "object": "entPhysicalIndex",
+            "implied": 0
+          },
+          {
+            "module": "ENTITY-MIB",
+            "object": "entPhysicalChildIndex",
+            "implied": 0
+          }
+        ],
+        "status": "current",
+        "description": "A single container/'containee' relationship."
+      },
+    ```
+    Sometimes indexes are columns from another table, and we might want to use another column as it could have more
+    human readable information - we might prefer to see the interface name vs its numerical table index. This can be
+    achieved using metric_tag_aliases
+
+
+    Return a list of SNMP metrics and copy its yaml dump to the clipboard
+    Metric tags need to be added manually
+    """
+    # ensure at least one mib file is provided
+    if len(mib_files) == 0:
+        print('🙄 no mib file provided, need at least one mib file to generate a profile')
+        return
+
+    # create a list of all mib files directories and mib names
+    source_directories = set()
+    mibs = set()
+    for file in mib_files:
+        source_directories.add(os.path.dirname(file))
+        mibs.add(os.path.splitext(os.path.basename(file))[0])
+    # create a tmp dir for compiled json mibs
+    json_destination_directory = os.path.join(gettempdir(), 'mibs')
+    if not os.path.exists(json_destination_directory):
+        os.mkdir(json_destination_directory)
+
+    profile_oid_collection = {}
+    # build profile
+    for oid_node in _extract_oids_from_mibs(list(mibs), list(source_directories), json_destination_directory, filters):
+        if oid_node.node_type == 'table':
+            _add_profile_table_node(profile_oid_collection, oid_node)
+        elif oid_node.node_type == 'row':
+            # requires
+            _add_profile_row_node(
+                profile_oid_collection,
+                oid_node,
+                os.path.dirname(mib_files[0]),
+                metric_tag_aliases_path=aliases,
+                json_mib_directory=json_destination_directory,
+            )
+        elif oid_node.node_type == 'column':
+            _add_profile_column_node(profile_oid_collection, oid_node)
+        elif oid_node.node_type == 'scalar':
+            _add_profile_scalar_node(profile_oid_collection, oid_node)
+
+    print('📋 Profile yaml:')
```

Comment:
I think we should not print debug messages like `Profile yaml:`, similar to other linux cmd tools.

The reason is that the `stdout` should be a valid yaml. You can possibly send debug prints to stderr.

Otherwise it won't work well with tools like `yq`.

Example:

```
§ ddev meta snmp generate-profile-from-mibs IF-MIB  | yq r - metrics[0]
MIB: IF-MIB
symbol:
  name: ifNumber
  OID: 1.3.6.1.2.1.2.1.0
  description: The number of network interfaces (regardless of their current state)
    present on this system.
```